packages/lib-classifier/src/store/SubjectViewerStore.js

```diff
@@ -28,7 +28,7 @@ const SubjectViewer = types
     - 'zoomValue' defines amount zoomed in/out, or current zoom value of 'zoomto'.
      */
     onZoom: function (type, zoomValue) {},
-    onPan: function (value) {}
+    onPan: function (type, value) {}
```

Comment:
I think this will break the TESS light curve viewer, so panning probably needs to be updated there too.

Reply:
Thanks for the heads up. We don't have a test covering this, so I didn't catch it.